Makefile

```diff
@@ -75,14 +75,17 @@ VERSION?=1.9.2
 IMAGE_TAG=$(REGISTRY_NAME)/$(IMAGE_NAME)
 IMAGE_TAG_DEV=$(REGISTRY_NAME)/$(IMAGE_NAME):latest-$(shell git rev-parse --short HEAD)
 
-docker: docker-official
+docker: docker-dev
 
-docker-light: export GOOS=linux
-docker-light: export GOARCH=amd64
-docker-light: export CGO_ENABLED=0
-docker-light:
+docker-dev-binary:
+	GOOS=linux \
+	GOARCH=amd64 \
+	CGO_ENABLED=0 \
+	go build -o bin/packer .
```

Comment:
For build I think we want to honor the GOLDFLAGS right?

Reply:
Good call, I'll add those